maintainers/maintainer-list.nix

```diff
@@ -6457,6 +6457,12 @@
     githubId = 757752;
     name = "Jonas Heinrich";
   };
+  onsails = {
+    email = "andrey@onsails.com";
+    github = "onsails";
+    githubId = 107261;
```

Comment:
Adding maintainer requires to be put in a separate commit per contributing guide.